tests/test_annotation_stores.py

```diff
@@ -1033,9 +1033,9 @@ def test_load_cases_error(fill_store, store_cls):
             store._load_cases(["foo"], lambda: None, lambda: None)
 
     @staticmethod
-    def test_py38_init(fill_store, store_cls, monkeypatch):
+    def test_py37_init(fill_store, store_cls, monkeypatch):
         """Test that __init__ is compatible with Python 3.8."""
```

Comment:
Minor comment inconsistency

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/391/commits/ecf47c6532b90ad72524838e9e65f0dd11f35f06